pymc/distributions/discrete.py

```diff
@@ -1411,6 +1412,23 @@ def logcdf(value, c):
         )
 
 
+class Constant:
```

Comment:
Can you add a test where you call `Constant` and confirm the warning is emitted and a DiracDelta is returned?